test/e2e/lib/driver-helper.js

```diff
@@ -233,8 +233,8 @@ export async function isElementPresent( driver, selector ) {
 	return !! elements.length;
 }
 
-export function elementIsNotPresent( driver, cssSelector ) {
-	return this.isElementPresent( driver, by.css( cssSelector ) ).then( function( isPresent ) {
+export function elementIsNotPresent( driver, selector ) {
+	return this.isElementPresent( driver, selector ).then( function( isPresent ) {
```

Comment:
The signature change here looks reasonable to me, but cc @stojdza @bsessions85 if y'all had thoughts. Btw did y'all need help with a new GH group handle for the Quality team?

Reply:
🤔 Scanning through the changeset I only see CSS selectors. @mmtr did you originally need an xpath selector?

Reply:
> Scanning through the changeset I only see CSS selectors. @mmtr did you originally need an xpath selector?

This change is to ensure that the selector passed to `elementIsNotPresent` is consistent with `isElementPresent` which expects a `By` object. 

That way, we can reuse `By` objects of both function as done [here](https://github.com/Automattic/wp-calypso/pull/38767/files#diff-77450bf032f03c9c64591ef8a0a8237eR163).

> Btw did y'all need help with a new GH group handle for the Quality team?

I think they already have @Automattic/e2e-test-reviewers.

Reply:
LGTM 👍